partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/contant/ClientServiceAuditEnum.java

```diff
@@ -0,0 +1,100 @@
+package io.mosip.pms.oidc.client.contant;
+
+import io.mosip.pms.partner.manager.constant.AuditConstant;
+
+public enum ClientServiceAuditEnum {
+
+    CREATE_CLIENT("PMS_PRT_100", AuditConstant.AUDIT_SYSTEM, "POST CREATE OIDC CLIENT",
```

Comment:
please check and try to give some consecutive numbers as event id

Reply:
Updated the Event ID's with the consecutive numbers